doc/Developer Guide/Test Step/Readme.md

```diff
@@ -334,7 +335,42 @@ For different approaches to publishing results, see the examples in:
 
 -	`TAP_PATH\Packages\SDK\Examples\PluginDevelopment\TestSteps\PublishResults`
 
-	
+## Child Test Steps
+
+Test step can have any number of child test steps. Exactly which can be controlled by using `AllowAnyChildAttribute`, and `AllowChildOfType`. 
+
+To execute child test steps within a test step run, the RunChildStep method can be used to run a single child step or RunChildSteps can be used to run all of them.
+
+Depending on the verdict of the child steps, the break condition setting of the parent step and the way RunChildSteps is called there are a number of different ways that the control flow will be affected.
+
+The default implementation should look something like this:
+```cs
+    public override void Run()
+    {
+        // Insert here things to do before running child steps 
+        RunChildSteps();
+        // Insert here things to do after running child steps.
+    }
+```
+
+`RunChildSteps` will take care of setting the verdict of the calling test step based on the verdict of the child test steps. The default algorithm here is to take the most severe verdict and use that for the parent test step. 
+For example, if the child test steps has verdict Pass, Inconclusive and Fail, the parent test step will get the Fail verdict.   
+
+`RunChildSteps` may throw an exception, this happens if the break conditions for the parent test step are satisfied. 
+If the break conditions are satisfied due to an exeception being thrown, resulting in an Error verdict, the same exception will be thrown.
+To avoid this behavior the overload `RunChildSteps(throwOnBreak: false)` can be used, but the exceptions are still available from the returned list of test step runs.
```

Comment:
```suggestion
To avoid this behavior, the overload `RunChildSteps(throwOnBreak: false)` can be used, but the exceptions are still available from the returned list of test step runs.
```